### ISSUE TITLE
Unable to yarn run dev:chrome for extension

### ISSUE THREAD

**Issue by nooma-42** [jzaki/bls-wallet#514]
I followed `local_development.md` to set up project and encountered following issue. 

```
➜  extension git:(main) ✗ yarn run dev:chrome
yarn run v1.22.19
warning ../../../package.json: No license field
$ cross-env NODE_ENV=development cross-env TARGET_BROWSER=chrome webpack --watch
[ Starting the Web Extension Hot Reload Server... ]
wext-manifest-webpack-plugin: removed js/manifest.bundle.js
(node:92437) [DEP_WEBPACK_CHUNK_HAS_ENTRY_MODULE] DeprecationWarning: Chunk.hasEntryModule: Use new ChunkGraph API
(Use `node --trace-deprecation ...` to show where the warning was created)
(node:92437) [DEP_WEBPACK_CHUNK_ENTRY_MODULE] DeprecationWarning: Chunk.entryModule: Use new ChunkGraph API
(node:92437) [DEP_WEBPACK_DEPRECATION_ARRAY_TO_SET] DeprecationWarning: chunk.files was changed from Array to Set (using Array method 'filter' is deprecated)
clean-webpack-plugin: pausing due to webpack errors
ERROR in Conflict: Multiple assets emit different content to the same filename js/home.bundle.js.map

ERROR in Conflict: Multiple assets emit different content to the same filename js/confirm.bundle.js.map

2023-02-08 23:06:01: webpack compiled (fc07d3c947b725908a3b)
No errors found.
```

This issue is suggested to be relevant with `webpack.config.js`, but there seems to be no problem in `extension/webpack.config.js`

```
output: {
    path: path.join(destPath, targetBrowser),
    filename: 'js/[name].bundle.js',
  },
```

**Comment by jacque006** [comment]
Even though there are these stated errors:

> ERROR in Conflict: Multiple assets emit different content to the same filename js/home.bundle.js.map
> 
> ERROR in Conflict: Multiple assets emit different content to the same filename js/confirm.bundle.js.map

It likely did not impact the final assets that were created for the extension.

> 2023-02-08 23:06:01: webpack compiled (fc07d3c947b725908a3b)
> No errors found.

Were you able to load the extension in browser? Or did you encounter a runtime error there when attempting to load/run the extension?

**Comment by nooma-42** [comment]
I were able to load the extension in browser. However, extension page seems fail to load onboarding art image, or is it a user custom image?
<img width="500" alt="截圖 2023-02-09 上午10 47 13" src="https://user-images.githubusercontent.com/35304302/217704803-30db79d0-8e2e-4682-aa4a-e2dfe92b2ee5.png">

-- edit
ok, I watched the demo video and the onboarding image appeared to be blank in default. Should I leave this issue open? (maybe some future work on `ERROR in Conflict: `) or should I just close it?
